### ISSUE TITLE
Android TV 

### ISSUE THREAD

**Issue by Fxrrcrxdsccexxe** [Zenda-Cross/vega-app#31]
Plz make Android TV compatible

**Comment by Zenda-Cross** [comment]
If I am able to integrate the existing app into react-native-tv using Expo without creating a new project, I will create a TV-compatible app.

**Comment by Sanjaydatt** [comment]
Plz make android tv compatible

**Comment by Zenda-Cross** [comment]
Update: I was able to run it on TVOS.

![image](https://github.com/Zenda-Cross/vega-app/assets/143804558/196eb1a7-2d90-4fe3-b794-dcd4050c8026)

**Comment by Fxrrcrxdsccexxe** [comment]
Plz make Android TV compatible as soon as possible give ur best...

**Comment by Oshsu2000** [comment]
@Zenda-Cross please make it 'android tv' compatible.
It'll be great. Please.

**Comment by Zenda-Cross** [comment]
> @Zenda-Cross please make it 'android tv' compatible. It'll be great. Please.

I will,  just not getting enough time to work on it

**Comment by Oshsu2000** [comment]
@Zenda-Cross thank you, appreciated ✌️

**Comment by LEOATIQ** [comment]
It is not installing now before it got installed when the version was 2.1.5 in mi tv can u fix the issue it is a really and some Turkish series doesn't it is showing that no stream found q

**Comment by Zenda-Cross** [comment]
For now, I can share a version that can be installed on all TVs but can't be controlled with the remote properly you may have to use a mouse.
What do you guys say?

**Comment by LEOATIQ** [comment]
Okk I am fine with that

**Comment by Fxrrcrxdsccexxe** [comment]
No plz me fully compatible with remote..

**Comment by Zenda-Cross** [comment]
Making remote compatible requires additional changes to every component/button

**Comment by LEOATIQ** [comment]
Ok make the mouse version

**Comment by Zenda-Cross** [comment]
https://github.com/Zenda-Cross/vega-app/releases/tag/v2.3.1-tv

**Comment by Fxrrcrxdsccexxe** [comment]
Is this tv version work with remote ..

**Comment by Zenda-Cross** [comment]
> Is this tv version work with remote ..

doesn't work properly

**Comment by LEOATIQ** [comment]
So I have to use wit mouse ryt

**Comment by Zenda-Cross** [comment]
> So I have to use wit mouse ryt

yeah

**Comment by LEOATIQ** [comment]
Okk thanks now I can use in tv 😊

**Comment by Sanjaydatt** [comment]
New version for tv work with mouse. thanks for UI changes👌

**Comment by LEOATIQ** [comment]
![20240819_195241.jpg](https://github.com/user-attachments/assets/3121f7b4-4488-4c7e-a8ac-54c37a14d1a6)

Why do I have to go settings and open the app

**Comment by Zenda-Cross** [comment]
> Why do I have to go to the setting to open the app

I don't know,  isn't there an option to add the app to the home screen?

**Comment by LEOATIQ** [comment]
No

**Comment by LEOATIQ** [comment]
I tried everything but no

**Comment by 123ccv95** [comment]
Hello, I'm running this on a projector with adroid 11 and runs excellent,  I think there is a little issue with the selector in the screen it's like if it were transparent, cause when I used with the remote can move but not with visibility, if you could fix this would be amazing, great app, for now using it with a mouse. and the full app version.

**Comment by Zenda-Cross** [comment]
Yes, the selector is not visible. I tried to fix that, but it didn't work. The only way is for me to handle the navigation myself, including all its movements with different keys, or use some external package. I am uncertain if I can do this

**Comment by Zenda-Cross** [comment]
> Hello, I'm running this on a projector with adroid 11 and runs excellent, I think there is a little issue with the selector in the screen it's like if it were transparent, cause when I used with the remote can move but not with visibility, if you could fix this would be amazing, great app, for now using it with a mouse. and the full app version.

Yes, the selector is not visible. I tried to fix that, but it didn't work. The only way is for me to handle the navigation myself, including all its movements with different keys, or use some external package. I am uncertain if I can do this

**Comment by dehffgs** [comment]
> https://github.com/Zenda-Cross/vega-app/releases/tag/v2.3.1-tv

hi. can you add support for old version android device? my android tv box still running on android 5.1 lollipop and the app cannot be installed. thank you :)

**Comment by Zenda-Cross** [comment]
> > https://github.com/Zenda-Cross/vega-app/releases/tag/v2.3.1-tv
> 
> hi. can you add support for old version android device? my android tv box still running on android 5.1 lollipop and the app cannot be installed. thank you :)

Sorry but that's not possible because some  the packages and libraries used by the app require a minimum Android version 7 to work
also, the tv app is not updated anymore

**Comment by abhaylahagora** [comment]
> > Why do I have to go to the setting to open the app
> 
> I don't know, isn't there an option to add the app to the home screen?

need to add leanback launcher in manifest to show app on TV

**Comment by NarendraY1** [comment]
Hi update this version too v2.3.1-tv(tv version)

**Comment by NarendraY1** [comment]
Installing and running Vega Android app into Android TV it works fine but when we play anything screen get Black out can you fix that video player in the app

**Comment by RayzaEFC** [comment]
Come on lets have a fully compatible Android TV version please! Great app, just needs to be fully operational for TV interface!

**Comment by princethakur33** [comment]
Copy some information from cloudstream and make app supported for Android tv/firestick

**Comment by abkoradiya** [comment]
@Zenda-Cross 
I have tried to make TV compatible. But it's very difficult to manage focus,
I can try to make this using native android if you agree.

**Comment by Fxrrcrxdsccexxe** [comment]
> > Tv app alpha build https://github.com/vega-org/vega-app/releases/tag/v3.2.6-tv-alpha-1
> 
> Seems to be working good 






Player not functioning properly like not able to do forward or backward video

**Comment by Suleman-Elahi** [comment]
Can't seek video. The menu Can't be closed without returning to episode selection. These are two major issues i have. I am happy with everything else. I am using on Google Chromecast.

**Comment by himanshu8443** [comment]
Anyone wants to use vega on TV use the new TV app, download it from here 

https://vega.8man.in/#desktop

https://github.com/vega-org/vega-desktop
